tests/release/README.md

```diff
@@ -56,4 +56,48 @@ This document will detail the e2e tests included for the release suite.
        - The release PipelineRun is successfully created in the managed namespace.
        - The release PipelineRun is expected to pass.
        - The Release passes.
-       - Validate that the SBOM for both components were uploaded to Pyxis stage.
\ No newline at end of file
+       - Validate that the SBOM for both components were uploaded to Pyxis stage.
+
+
+### 5. Negative e2e-tests
+We use manual tests to cover negative tests, using filtering labels in Ginkgo framework to enable users to run negative e2e-tests whether all together or each one separately:
+
+
+
+**Run all negative release e2e-tests**
+```bash
+./bin/e2e-appstudio --ginkgo.junit-report=report.xml  --ginkgo.focus="release" --ginkgo.label-filter="release-neg" --ginkgo.vv
+```
+
+**Run all negative release e2e-tests**
+```bash
+e2e-test = negMissingReleaseStrategy || negMissingReleasePlan || negMissingReleasePlanAdmission
+./bin/e2e-appstudio --ginkgo.junit-report=report.xml  --ginkgo.focus="release" --ginkgo.label-filter="<e2e-test>" --ginkgo.vv
+```
+
+
+   #### 5.1. A release CR will fail if ReleasePlanAdmission is missing(neg-missing-releaseplanadmission.go).
+
+      Steps:
+       - Create the following CRs: Snapshot, ReleaseStartegy, ReleasePlan, EnterpriseContractPolicy and Release          
+      Expected Results:
+       - A release CR is created.
+       - We ensure that Release CR is Failed on Validation and on Release, as a proper message printed out to user. 
```

Comment:
`s/is Failed/fails/`
`s/, as a proper message printed out to/ with a proper message printed to the/`

Reply:
We ensure that Release CR fails on Validation and on Release, with a proper message printed out to the user.

Reply:
Still not fixed

Reply:
This comment is still not addressed